src/cmps/common/ConfettiAnimation.vue

```diff
@@ -17,14 +17,30 @@
 </template>
 
 <script>
-export default {}
+export default {
+  data() {
+    return {
+      isAnimationstarted: false,
+    }
+  },
+
+  mounted() {
+    setTimeout(() => {
```

Comment:
תבדוק אם צריך לנקות setTimout כמו שצריך setInterval

Reply:
isAnimationstarted should be isAnimationStarted, and maybe isAnimationRunning is a better name?

Reply:
בדקתי וזה לא אמור לשנות אין הכרח כמו בsetInterval